pymc/logprob/order.py

```diff
@@ -81,12 +81,12 @@ def find_measurable_max(fgraph: FunctionGraph, node: Node) -> Optional[List[Tens
     # Non-univariate non-RVs must be rejected
     if (
         not isinstance(base_var.owner.op, RandomVariable)
-        and base_var.owner.inputs[0].owner.op.ndim_supp == 0
+        and base_var.owner.inputs[0].owner.op.ndim_supp != 0
```

Comment:
On changing both to not, our test fails to reject non-rvs such as " x = pt.exp(pt.random.beta(0, 1, size=(3,))) " as the ndim_supp in this case comes out tp be equal to zero, hence the None statement is never encountered.

Reply:
I don't remember my original suggestion but the idea was `if not (rv and ndim_supp==0): return None`

Reply:
Originally I think we planned to reject any Non-RVs.
By this logic, ```x = pt.exp(pt.random.beta(0, 1, size=(3,)))``` does not fail as though exp (being elemwise) is a non rv, a beta distribution with size = (3, ) does not meet the above condition. Should I update the test to something like ```x = pt.exp(pt.random.beta(0, 1, size=(3, 3)))``` so that it meets the condition?

Reply:
Oh I was not clear, the point was to remove multivariate RVs which I think the rewrite doesn't reject. For instance StickBreakingWeights is a multivariate RV with scalar inputs.